tycho-indexer/src/substreams/stream.rs

```diff
@@ -144,7 +163,7 @@ fn stream_blocks(
                     // We failed to connect and will try again; this is another
                     // case where we actually _want_ to back off in case we keep
                     // having connection errors.
-
+                    counter!("substreams_failure", "module" => output_module_name.clone(), "cause" => "connection_error").increment(1);
```

Comment:
Some metrics you add a "total" in the end, like (rpc_requests_total) - and other you leave it out - is there a reason? I'd like to have a pattern here

Reply:
I actually started to remove them from the metric names as it was making them long without adding any info. I missed this one I suppose.